instrumentation/net/http/otelhttp/transport_test.go

```diff
@@ -279,6 +279,10 @@ func TestWrappedBodyClose(t *testing.T) {
 	wb := &wrappedBody{span: trace.Span(s), body: readCloser{}}
 	assert.NoError(t, wb.Close())
 	s.assert(t, true, nil, codes.Unset, "")
+
+	var body io.ReadCloser
```

Comment:
How about making this its own test (like the following one, `TestWrapperBodyCloseError`)?
It'd make it easier to see why a failure happens.

Reply:
Yeah, I think it is better.

Reply:
Updated.